src/random.h

```diff
@@ -12,6 +12,14 @@
 
 #include <stdint.h>
 
+#ifdef _MSC_VER
```

Comment:
This should likely go into a compatiblity header?
I wonder why the `ssize_t` in `netbase.cpp` doesn't trip it up.